java/arcs/android/devtools/DevWebSocket.kt

```diff
@@ -0,0 +1,54 @@
+package arcs.android.devtools
+
+import android.content.ContentValues.TAG
+import android.util.Log
+import fi.iki.elonen.NanoHTTPD
+import fi.iki.elonen.NanoWSD
+import java.io.IOException
+
+class DevWebSocket : NanoWSD(12345) {
+    var wsd: WsdSocket? = null
+    protected override fun openWebSocket(ihttpSession: NanoHTTPD.IHTTPSession?): WebSocket {
```

Comment:
```suggestion
    private var wsd: WsdSocket? = null
    
    protected override fun openWebSocket(ihttpSession: NanoHTTPD.IHTTPSession?): WebSocket {
```

Reply:
Also, `wsd` is a bit too terse.

Reply:
Also also, I think the public methods should go above the protected/private ones.

Reply:
I moved things around and updated the name.